CONTRIBUTING.md

```diff
@@ -30,6 +36,54 @@
     chore: update OIDC script
     ```
 
-1. Create a pull request to merge your changes into the `main` branch (or the relevant `release/*` branch if you're making a change to a previous major version).
+1. Create a pull request to merge your changes into the `main` branch.
 
     Use the Conventional Commits specification for semantic pull request titles.
+
+## 🤝 Roles and responsibilities
+
+This section describes the various roles and responsibilities in the Ops Actions project.
+
+### 🦸‍♀️ External contributors
+
+An external contributor should:
+
+- Create forks of this repository.
+- Create issues and pull requests in this repository.
+
+### 👨‍🎓 Contributors
+
+A contributor must:
+
+- Be a member of the [@equinor/ops-actions](https://github.com/orgs/equinor/teams/ops-actions) team.
+- Actively contribute to this repository.
+
+A contributor should:
+
+- Have basic understanding of GitHub Actions.
+- Have basic understanding of our best practices.
+
+### 👷‍♀️ Maintainers
+
+A maintainer must:
+
+- Be a member of the [@equinor/ops-actions-maintainers](https://github.com/orgs/equinor/teams/ops-actions-maintainers) team.
+- Review and approve pull requests in this repository.
+- Discuss and update our best practices when needed.
+
+A maintainer should:
+
+- Have complete understanding of GitHub Actions.
```

Comment:
Instead of `complete understanding`, perhaps use `advanced understanding`? Nitpicking, I know...

Reply:
Good point 🙌 

An argument of keeping it as is: this requirement is listed under "A maintainer **should**" instead of "A maintainer **must**". IMO, the word "should" implies that a maintainer must **strive to achieve complete understanding**, however it's not a requirement per se.

Reply:
Yes, I see your point. On the other hand, it might create false expectations to the maintainers, and make it difficult to recruit people for the role.

I don't know.. I can live with both wordings, but I thought it's worth discussing. 😁

Reply:
I'll update the wording an re-request a review from you 👍